src/index.ts

```diff
@@ -22,10 +22,8 @@ import * as v1beta1 from './v1beta1';
 const DataCatalogClient = v1.DataCatalogClient;
 type DataCatalogClient = v1.DataCatalogClient;
 const PolicyTagManagerClient = v1beta1.PolicyTagManagerClient;
-type PolicyTagManagerClient = v1beta1.PolicyTagManagerClient;
```

Comment:
@alexander-fenster, these types are being deleted when the library is regenerated, which breaks the system-test. ANy idea why/should they be getting deleted?